plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# check the certificate validity of your certfificates
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Installing: Add list of your certificates prefixed by the type in munin plugin-conf.d
+#
+# For openvpn ca.crt and crl.pem
+# [cert_validity]
+# user root
+# env.CERTS crl:/etc/openvpn/easy-rsa/keys/crl.pem x509:/etc/openvpn/easy-rsa/keys/ca.crt
+#
+# For letsencrypt certficates
+# [cert_validity]
+# user root
+# env.CERTS x509:/etc/letsencrypt/live/domain1.example.com/cert.pem x509:/etc/letsencrypt/live/domain2.example.com/cert.pem
+#
+# Dependencies: openssl
+#
+# Parameters:
+#
+#  config
+
+
+if [ "$1" == "config" ] ; then
+	echo "graph_title Certificate validity"
+	echo "graph_args --logarithmic --base 1000"
+	echo "graph_vlabel certificate validity in days"
+	echo "graph_category certificates"
+fi
+
+now=$(date +%s)
+for cert in ${CERTS[@]}; do
+	cert_type=${cert%:*}
+	cert_file=${cert#*:}
+	cert_name=${cert_file//\//_}
```

Comment:
Please use `clean_fieldname "$cert_file"` instead.

(you need to add `. "$MUNIN_LIBDIR/plugins/plugin.sh"` before)

Reply:
changed to `clean_fieldname $cert_file`